framework/src/main/java/framework/java/measurement/MethodReturnVisitor.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * The MIT License (MIT)
+ * Copyright (c) 2017 FI MUNI
+ */
+package main.java.framework.java.measurement;
+
+import org.sonar.plugins.java.api.tree.*;
+
+import java.util.List;
+
+/**
+ * Class responsible for method's return type retrieval.
+ * @author Filip Cekovsky
+ */
+public class MethodReturnVisitor extends BaseTreeVisitor{
+
+    /** Package of the visited method */
+    private String packageName;
+    /** Eventual contains fully qualified name of the owner class*/
+    private String identifier = "";
+    /** Imports present in the class */
+    private List<String> imports;
+
+    /**
+     * Constructor of the class
+     * @param packageName fills {@link #packageName}
+     * @param imports fills {@link #imports}
+     */
+    public MethodReturnVisitor(String packageName, List<String> imports){
+        this.packageName = packageName;
+        this.imports = imports;
+    }
+
+    @Override
+    public void visitClass(ClassTree tree){
+        identifier = MeasurementUtils.getClassName(tree, packageName);
+    }
+
+    @Override
+    public void visitMethod(MethodTree tree){
+        if (tree.returnType() == null){
```

Comment:
ake pripady toto pokryva? nejak si to nedokazem predstavit

Reply:
Vytiahne to text returnu metody, potom sa pokusi zistit, ci je returnovana metoda v importoch alebo to nahodou nie je ta v ktorej sa nachadzame, ak ano tak zobere cely ten import (co je vlastne fully qualified name), ak nie, vezme co nasiel v dekleracii metody a ulozi iba to. Tie nie fully qualified potom dalej spracovavam vo svojom plugine ...

Reply:
Jaj uz rozumiem o co slo, to su konstruktory, ktore nemaju ako vratnu hodnotu ani hodnotu void ale vysledkom je vlastne instancia tej triedy.